### PR TITLE
set myself depending on participantId

### PR DESCRIPTION
**Pull request by Qanah**
Hay Guys

I try to make the component work automatically without add extra code from the user

```
newMessage: (state, message) => {
    message.timestamp = moment(message.timestamp);
    message.myself = message.participantId === state.myself.id;
    state.messages = [...state.messages, message];
}

setMessages: (state, messages) => {
    messages.map(message => {
        message.timestamp = moment(message.timestamp);
        message.myself = message.participantId === state.myself.id;
    });
    state.messages = messages;
}
```

### CODE REVIEW

File: src/store.js [hunk 1]
```diff
@@ -42,21 +44,15 @@ export default () => {
...
-                    if (participant.id == id) {
+                    if (participant.id === id) {
                         curr_participant = participant;
                     }
                 });
 
                 return curr_participant;
             },
             messages: (state) => {
-                let messages = [];
```
Comment (MatheusrdSantos) [inline src/store.js]: Those lines can't be removed. They are important to keep the component easy to debug with Vuex Timetravel.
Please, check this issue #6 that explains the problem and this PR #7 that fixes the bug.

File: src/store.js [hunk 2]
```diff
@@ -15,7 +15,8 @@ export default () => {
         },
         mutations: {
             newMessage: (state, message) => {
-                message.timestamp = message.timestamp.toISOString();
```
Comment (MatheusrdSantos) [inline src/store.js]: This line can't be removed. It's explained in the comment below.

File: src/store.js [hunk 3]
```diff
@@ -42,21 +44,15 @@ export default () => {
...
                 return curr_participant;
             },
             messages: (state) => {
-                let messages = [];
-                state.messages.forEach(message => {
-                    let newMessage = {...message};
-                    newMessage.timestamp = moment(newMessage.timestamp);
-                    messages.push(newMessage);
-                });
-                return messages;
```
Comment (MatheusrdSantos) [inline src/store.js]: Those lines can't be removed. They are important to keep the component easy to debug with Vuex Timetravel.
Please, check this issue #6 that explains the problem and this PR #7 that fixes the bug.

**Comment by MatheusrdSantos** [comment]
Hi, @Qanah! I agree with your changes.
Please, fix the PR as I commented. After the fixes, I'll merge your PR.

**Comment by Qanah** [comment]
Hi, @MatheusrdSantos Please check now.

**Comment by MatheusrdSantos** [comment]
@Qanah, thanks for your contribution.

**Pull request merged**